IPython/html/base/handlers.py

```diff
@@ -250,6 +255,7 @@ def template_namespace(self):
             sys_info=sys_info,
             contents_js_source=self.contents_js_source,
             version_hash=self.version_hash,
+            **self.jinja_template_vars
```

Comment:
If you use `ns.update(self.jinja_template_vars)` instead of `**vars`, you will be able to override existing values. Do you think we should allow that?